backend/beyond_the_loop/routers/models.py

```
@@ -135,11 +134,51 @@ async def toggle_model_by_id(id: str, user=Depends(get_verified_user)):
+    bookmarked =Models.toggle_bookmark(model.id, user.id)
+    return {"model_id": model.id, "bookmarked_by_user": bookmarked}
 
 ############################
 # UpdateModelById
 ############################
 
+@router.post("/model/update", response_model=Optional[ModelModel])
+async def update_model_by_id(
+    id: str,
+    form_data: ModelForm,
+    user=Depends(get_verified_user),
```

Ruff (0.13.1)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)